cqrs-user-command-service/src/integrationTest/java/dev/amir/usercommand/util/DefaultObject.java

```diff
@@ -5,14 +5,15 @@
 import dev.amir.usercommand.domain.valueobject.UserId;
 import dev.amir.usercommand.domain.valueobject.UserPassword;
 import dev.amir.usercommand.domain.valueobject.UserStatus;
+import dev.amir.usercommand.domain.valueobject.UserUsername;
 import dev.amir.usercommand.framework.output.sql.entity.UserJpa;
 import java.time.LocalDate;
 import java.time.LocalDateTime;
 
 public final class DefaultObject {
     public static final UserId defaultUserId = new UserId("b6a3d07d-0c0d-4d7f-b968-453808256e31");
     public static final RoleId defaultRoleId = new RoleId("9abef656-906f-11ee-b9d1-0242ac120002");
-    public static final String defaultUsername = "user123abc";
+    public static final UserUsername defaultUsername = new UserUsername("user123abc");
```

Comment:
on this line, why is it called defaultUsername and not defaultUserUsername? it's confusing